### PR TITLE
chore: opbnb

### PR DESCRIPTION
**Pull request by chefjackson**
### Summary
🚀🌉💸

This pull request adds support for the opBNB Mainnet, a layer-2 chain that runs on Binance Smart Chain, to various features and packages of the PancakeSwap app and ecosystem. It updates the chain configuration, contract addresses, token information, and data fetching logic for the opBNB chain in files such as `chains.ts`, `contracts.ts`, `exchange.ts`, `nodes.ts`, `pools.ts`, `common.ts`, `gasModel`, `v3.ts`, `constants.ts`, `NetworkSwitcher.tsx`, `pancake-default.tokenlist.json`, `fetchChunk.ts`, `fetchDerivedPriceData.ts`, `blockConflictTolerance.ts`, `multicall.ts`, and `onChainQuoteProvider.ts`. It also fixes some minor issues and adds comments for clarity.

> _We're breaking the chains of the Binance Smart Chain_
> _We're rising to the layer-2 with opBNB Mainnet_
> _We're swapping and routing with the smartest contracts_
> _We're burning the gas with the wrapped BNB token_

### Walkthrough
*  Add support for opBNB Mainnet, a layer-2 chain that uses Binance Smart Chain as its base layer ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-3ff81a954673aa998f9490c76adc346de7fb57b7b526e9bcddbc9cd7cc12af6cR19), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-5b02b946734e39568c018763a8445fe22e5633c04fa393f78eaec1947165713dR154), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4R34), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4L78-R79), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4L93-R94), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4L105-R135), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4R191), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4R212), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07R35), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07R207), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07R227), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR16), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR45), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR66), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR93), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR120), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-55ee773d94e57d4fe20f6463bc02307f5d9747933c77b517fa31a90375d2af7eR422-R429), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-17218d64565ce12e09953351575126053d6b30541c3605f7ee38a34374a7daecL14-R14), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-17218d64565ce12e09953351575126053d6b30541c3605f7ee38a34374a7daecR59), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-17218d64565ce12e09953351575126053d6b30541c3605f7ee38a34374a7daecR105), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ddce95e802c6a0d90c11aede57b78c496812dc5c4b41ad90624e555af6bde95R17), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ddce95e802c6a0d90c11aede57b78c496812dc5c4b41ad90624e555af6bde95R38), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ddce95e802c6a0d90c11aede57b78c496812dc5c4b41ad90624e555af6bde95R58), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-756d218de2219271e4b96affb244c02cc9c25021b88c7e5bc41aa07479009f3fR14), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-dfa0d43a8db000e0c6466316afa72d83e8ce626238b0960dae8ecdf2de43bfc3R58), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-44b63d5a37b209699e3905878081df3ed1a2f5022176751c0bf316e159d27cc0R13), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-2da858338df7cb5c96454014265a897661a67524036f7b7645b14812c94fb86aR12), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-49573f110ec3a7b2c18b2aec89dcca7184192e3187731aaed499af0da5ae3c08R11-R12), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-49573f110ec3a7b2c18b2aec89dcca7184192e3187731aaed499af0da5ae3c08R26), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R19), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R40), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R60), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R80), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R109), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-24d3b14038aadecce9558c47648b19363b226e949e3580a1381f1b4eb5af9bdfR17), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-24d3b14038aadecce9558c47648b19363b226e949e3580a1381f1b4eb5af9bdfR35), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-547db0075f559af065060c380ed030a74d1c823c7062ee7e8926f73abeb3c24aR16), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-547db0075f559af065060c380ed030a74d1c823c7062ee7e8926f73abeb3c24aR33), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-547db0075f559af065060c380ed030a74d1c823c7062ee7e8926f73abeb3c24aR51), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-811cce7aeac21c9b1e6aceba276b6f1c541076dbf4722a255a0c1c7c40b9c3ccR68), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-b61d8bdd98a286c72deb45821a4ec9c7f76e5d799076e05a8f094a0ca72f79c2R23), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-b61d8bdd98a286c72deb45821a4ec9c7f76e5d799076e05a8f094a0ca72f79c2R43), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-9ee18cb7fa7b08aada0e9a3c92dede9202fb872ba10bd8f230843faef707f463R31), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R13), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R45), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R68), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R239-R246), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R263), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595L258-R279), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R297))
  * Specify the chain id, name, network, native currency, rpcUrls, blockExplorers, contracts, and testnet properties for the opBNB Mainnet in `apps/web/src/config/chains.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4L105-R135))
  * Add the opBNB Mainnet entry to the `evmChains` array in `apps/aptos/components/NetworkSwitcher.tsx` to enable switching to that chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-3ff81a954673aa998f9490c76adc346de7fb57b7b526e9bcddbc9cd7cc12af6cR19))
  * Add the opBNB Mainnet entry to the `SHORT_SYMBOL` object in `apps/web/src/components/NetworkSwitcher.tsx` to display the symbol 'opBNB' for that chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-5b02b946734e39568c018763a8445fe22e5633c04fa393f78eaec1947165713dR154))
  * Add the opBNB Mainnet entry to the `CHAIN_QUERY_NAME` object in `apps/web/src/config/chains.ts` to use the query name 'opBnb' for URL parameters ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4R34))
  * Replace the hard-coded chain id with the `ChainId.OPBNB_TESTNET` enum value for the opBNB Testnet entry in `apps/web/src/config/chains.ts` to make the code more consistent and maintainable ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4L78-R79))
  * Update the URL for the default block explorer for the opBNB Testnet entry in `apps/web/src/config/chains.ts` to use the testnet subdomain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4L93-R94))
  * Add the opBNB Mainnet entry to the `L2_CHAINS` array and the `CHAINS` array in `apps/web/src/config/chains.ts` to include it in the supported layer-2 chains and the supported chains ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4R191), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-870705bc610c873933b2d04d9a57db54345cab8d9ad2af707454ac577b21f6e4R212))
  * Add the addresses of the Multicall3, WAGMI token, and WAGMI vault contracts for the opBNB Mainnet in `apps/web/src/config/constants/contracts.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07R35), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07R207), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07R227))
  * Import the `opBnbTokens` object from `apps/web/src/config/constants/tokens.ts` in various files to access the information about the tokens that are available on the opBNB Mainnet chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR16), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R19), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-24d3b14038aadecce9558c47648b19363b226e949e3580a1381f1b4eb5af9bdfR17))
  * Add the opBNB Mainnet entry to the `CHAIN_REFRESH_TIME` object in `apps/web/src/config/constants/exchange.ts` to set the refresh time interval for fetching data from the chain to 6 seconds ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR45))
  * Add the opBNB Mainnet entry to the `SUGGESTED_BASES` object, the `BASES_TO_TRACK_LIQUIDITY_FOR` object, and the `PINNED_PAIRS` object in `apps/web/src/config/constants/exchange.ts` to use the wrapped BNB token as the suggested base token, the base token for tracking liquidity, and the default pair for swapping on that chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR66), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR93), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6c1ab0c7f31ca2b1900f09bca08f47eb81d6df582ebc368bdcc1000dbf59659fR120))
  * Add the wrapped BNB token entry for the opBNB Mainnet in `apps/web/src/config/constants/tokenLists/pancake-default.tokenlist.json` to include it in the supported tokens by the PancakeSwap exchange ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-55ee773d94e57d4fe20f6463bc02307f5d9747933c77b517fa31a90375d2af7eR422-R429))
  * Import the `opbnb` object from `apps/web/src/config/chains.ts` in `apps/web/src/config/nodes.ts` to access the information about the opBNB Mainnet chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-17218d64565ce12e09953351575126053d6b30541c3605f7ee38a34374a7daecL14-R14))
  * Add the opBNB Mainnet entry to the `SERVER_NODES` object and the `PUBLIC_NODES` object in `apps/web/src/config/nodes.ts` to use the public RPC URLs provided by the `opbnb` object for the app's server and client ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-17218d64565ce12e09953351575126053d6b30541c3605f7ee38a34374a7daecR59), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-17218d64565ce12e09953351575126053d6b30541c3605f7ee38a34374a7daecR105))
  * Add the opBNB Mainnet entry to the `POOLS_FAST_REVALIDATE` object, the `POOLS_NORMAL_REVALIDATE` object, and the `POOLS_SLOW_REVALIDATE` object in `apps/web/src/config/pools.ts` to set the revalidation time intervals for fetching data from the pools that have fast, normal, and slow updates to 10, 15, and 20 seconds respectively ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ddce95e802c6a0d90c11aede57b78c496812dc5c4b41ad90624e555af6bde95R17), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ddce95e802c6a0d90c11aede57b78c496812dc5c4b41ad90624e555af6bde95R38), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-6ddce95e802c6a0d90c11aede57b78c496812dc5c4b41ad90624e555af6bde95R58))
  * Add the opBNB Mainnet entry to the `l2DifferentBlockNumberChains` array in `apps/web/src/state/multicall/fetchChunk.ts` to adjust the multicall logic for fetching data from different chains ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-756d218de2219271e4b96affb244c02cc9c25021b88c7e5bc41aa07479009f3fR14))
  * Add the opBNB Mainnet entry to the `SWAP_INFO_BY_CHAIN` object in `apps/web/src/state/swap/fetch/fetchDerivedPriceData.ts` and assign it an empty object to indicate that the swap feature is not supported on that chain yet ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-dfa0d43a8db000e0c6466316afa72d83e8ce626238b0960dae8ecdf2de43bfc3R58))
  * Add the opBNB Mainnet entry to the `BLOCK_CONFLICT_TOLERANCE` object in `packages/multicall/src/constants/blockConflictTolerance.ts` and assign it a number of 3 to indicate the tolerance for block conflicts when using multicall ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-44b63d5a37b209699e3905878081df3ed1a2f5022176751c0bf316e159d27cc0R13))
  * Add the opBNB Mainnet entry to the `MULTICALL_ADDRESSES` object in `packages/multicall/src/constants/contracts.ts` and specify the address of the deployed Multicall contract on that chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-2da858338df7cb5c96454014265a897661a67524036f7b7645b14812c94fb86aR12))
  * Add a comment to the `common.ts` file in `packages/smart-router/evm/constants` to provide links to the sources that are used to map chain ids to coingecko ids ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-49573f110ec3a7b2c18b2aec89dcca7184192e3187731aaed499af0da5ae3c08R11-R12))
  * Add the opBNB Mainnet entry to the `CHAIN_ID_TO_CHAIN_NAME` object in `packages/smart-router/evm/constants/common.ts` and assign it an empty string to indicate that the chain name is not available yet ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-49573f110ec3a7b2c18b2aec89dcca7184192e3187731aaed499af0da5ae3c08R26))
  * Add the opBNB Mainnet entry to the `SMART_ROUTER_ADDRESSES` object, the `V2_ROUTER_ADDRESS` object, and the `STABLE_SWAP_INFO_ADDRESS` object in `packages/smart-router/evm/constants/exchange.ts` and specify the addresses of the deployed Smart Router, V2 Router, and Stable Swap Info contracts on that chain, or assign an empty string if the feature is not supported ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R40), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R60), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R80))
  * Add the opBNB Mainnet entry to the `BASES_TO_CHECK_TRADES_AGAINST` object in `packages/smart-router/evm/constants/exchange.ts` and use the wrapped BNB token as the base token for checking trades against ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-78ce9cd83b6e1807ee9752c395d004a76077b4df666c24beafa7932c76193b02R109))
  * Add the opBNB Mainnet entry to the `usdGasTokensByChain` object in `packages/smart-router/evm/constants/gasModel/index.ts` and use the wrapped BNB token as the token for estimating the gas cost in USD ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-24d3b14038aadecce9558c47648b19363b226e949e3580a1381f1b4eb5af9bdfR35))
  * Add the opBNB Mainnet case to the `BASE_SWAP_COST_V3` function, the `COST_PER_INIT_TICK` function, and the `COST_PER_HOP_V3` function in `packages/smart-router/evm/constants/gasModel/v3.ts` and return the same values as the Binance Smart Chain case ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-547db0075f559af065060c380ed030a74d1c823c7062ee7e8926f73abeb3c24aR16), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-547db0075f559af065060c380ed030a74d1c823c7062ee7e8926f73abeb3c24aR33), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-547db0075f559af065060c380ed030a74d1c823c7062ee7e8926f73abeb3c24aR51))
  * Add the opBNB Mainnet entry to the `BATCH_MULTICALL_CONFIGS` object in `packages/smart-router/evm/constants/multicall.ts` and assign it the `DEFAULT` object for the batch multicall configuration ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-811cce7aeac21c9b1e6aceba276b6f1c541076dbf4722a255a0c1c7c40b9c3ccR68))
  * Add the opBNB Mainnet entry to the `MIXED_ROUTE_QUOTER_ADDRESSES` object and the `V3_QUOTER_ADDRESSES` object in `packages/smart-router/evm/constants/v3.ts` and specify the addresses of the deployed Mixed Route Quoter and V3 Quoter contracts on that chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-b61d8bdd98a286c72deb45821a4ec9c7f76e5d799076e05a8f094a0ca72f79c2R23), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-b61d8bdd98a286c72deb45821a4ec9c7f76e5d799076e05a8f094a0ca72f79c2R43))
  * Add the opBNB Mainnet entry to the `SUCCESS_RATE_CONFIG` object in `packages/smart-router/evm/v3-router/providers/onChainQuoteProvider.ts` and assign it a number of 0.1 to indicate the success rate threshold for the smart router ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-9ee18cb7fa7b08aada0e9a3c92dede9202fb872ba10bd8f230843faef707f463R31))
  * Add the opBNB Mainnet entry to the `ChainId` enum, the `FACTORY_ADDRESS_MAP` object, the `INIT_CODE_HASH_MAP` object, the `WBNB` object, the `WNATIVE` object, and the `NATIVE` object in `packages/swap-sdk/src/constants.ts` and specify the chain id, the addresses of the Uniswap V2 factory and pair contracts, the ERC20Token instances of the wrapped BNB and native tokens, and the information of the native token for that chain ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R13), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R45), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R68), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R239-R246), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R263), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595L258-R279), [link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595R297))
  * Extract the information of the BNB token into a separate constant, `BNB`, in `packages/swap-sdk/src/constants.ts` and use it for both the Binance Smart Chain and the opBNB Mainnet entries in the `NATIVE` object to make the code more concise and consistent ([link](https://github.com/pancakeswap/pancake-frontend/pull/7838/files?diff=unified&w=0#diff-47643406d29b8598e2ec7264a59eca0794e0b4f77eded976f680d8dc5c145595L258-R279))

### CODE REVIEW

**Review by ChefJerry — APPROVED** [review]

File: packages/tokens/src/common.ts [hunk 1]
```diff
@@ -331,6 +331,13 @@ export const USDT = {
     'USDT',
     'Tether USD',
   ),
+  [ChainId.OPBNB]: new ERC20Token(
+    ChainId.OPBNB,
+    '0x9e5aac1ba1a2e6aed6b32689dfcf62a509ca96f3',
```
Comment (chef-jojo) [inline packages/tokens/src/common.ts:336]: Better checksum to avoid warning

File: packages/swap-sdk/src/constants.ts [hunk 2]
```diff
@@ -156,6 +158,14 @@ export const WETH9 = {
     'Wrapped Ether',
     'https://weth.io'
   ),
+  [ChainId.OPBNB]: new ERC20Token(
+    ChainId.OPBNB,
+    '0xe7798f023fc62146e8aa1b36da45fb70855a77ea',
```
Comment (chef-jojo) [inline packages/swap-sdk/src/constants.ts:163]: Ditto

File: apps/web/src/config/constants/lists.ts [hunk 3]
```diff
@@ -25,6 +26,10 @@ const ZKSYNC_URLS = [PANCAKE_ZKSYNC_DEFAULT, 'https://tokens.coingecko.com/zksyn
 const OP_SUPER_CHAIN_URL =
   'https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/master/optimism.tokenlist.json'
 const BASE_URLS = [PANCAKE_BASE_DEFAULT, OP_SUPER_CHAIN_URL, 'https://tokens.coingecko.com/base/all.json']
+const OPBNB_URLS = [
+  PANCAKE_OPBNB_DEFAULT,
+  'https://raw.githubusercontent.com/bnb-chain/opbnb-bridge-tokens/main/opbnb.tokenlist.json',
```
Comment (chef-jojo) [inline apps/web/src/config/constants/lists.ts:31]: Note: cake token is under determined

**Comment by chef-jojo** [comment]
Remind to add changeset

File: apps/web/src/config/constants/lists.ts (hunk 3 above)
Comment (chef-jojo) [reply apps/web/src/config/constants/lists.ts:31]: Let's remove it for now...

Comment (chefjackson) [reply apps/web/src/config/constants/lists.ts]: removed

**Pull request merged**